apps/site/src/views/Home/sections/FAQ/FAQ.tsx

```diff
@@ -1,11 +1,63 @@
+import { useEffect, useState } from "react";
```

Comment:
Issue: We should be using [`next-sanity`](https://github.com/sanity-io/next-sanity) instead of `useEffect` and `useState`.